crates/bindings-csharp/Runtime/Runtime.csproj

```diff
@@ -11,6 +11,7 @@
     <TargetFramework>net8.0</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <RootNamespace>SpacetimeDB</RootNamespace>
+    <Nullable>enable</Nullable>
```

Comment:
This is already enabled for all projects in `Directory.Build.props`, not necessary here.